components/logins/android/src/main/java/mozilla/appservices/logins/rust/PasswordSyncAdapter.kt

```diff
@@ -89,6 +90,12 @@ internal interface PasswordSyncAdapter : Library {
     fun sync15_passwords_update(handle: LoginsDbHandle, existing_login_json: String, error: RustError.ByReference)
 
     fun sync15_passwords_destroy_string(p: Pointer)
+
```

Comment:
I've added these here to make sure that this stays up to date with what's exposed from rust, even though we won't use them yet.